stats/statsd_test.go

```diff
@@ -421,6 +422,10 @@ func TestStatsdRegisterCollector(t *testing.T) {
 			if len(received) != len(expected) {
 				return false
 			}
+
+			sort.Strings(received)
+			sort.Strings(expected)
+
```

Comment:
Have you tried this?

```suggestion

			require.ElementsMatch(t, received, expected)
```

Reply:
Sorry didn't realize we're inside a `require.Eventually` so we can't use another require :+1: